### ISSUE TITLE
[Issue]: Jellyfin server flatpak is outdated and the client will not work android TV

### ISSUE THREAD

**Issue by gopikrishnansa** [jellyfin/jellyfin#12265]
### Please describe your bug

If we load the android tv, and if the server from the flatpak is 10.8, it shows that 10.8 will not work from the next version. If there is any way please update the flatpak package, we are not tech savy and configuring flatpak was very easy.

### Reproduction Steps

1. Install jellyfin server from flatpak
2. open jellyfin app from android tv

### Jellyfin Version

10.8.13

### Environment

```markdown
- OS:Ubuntu
- Linux Kernel:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
warning msg
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gopikrishnansa** [comment]
No, the latest version available on jellyfn server flatpak is 10.8

**Comment by felix920506** [comment]
Jellyfin flatpak is not maintained by us.
